composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/Descriptor.kt

```diff
@@ -22,31 +22,18 @@ data class Descriptor(
     val expirationDate: LocalDateTime?,
     val netTests: List<NetTest>,
     val longRunningTests: List<NetTest> = emptyList(),
-    val source: Source,
+    val source: InstalledTestDescriptorModel,
```

Comment:
I had mentioned this before, but keeping both the `Descriptor` and the `InstalledTestDescriptorModel` is just a big duplication of fields now. If we are to keep both because it helps with `UpdateStatus`, then we should remove the duplication:
- The `Descriptor` class should just have the `source` and `UpdateStatus`
- Optionally, `InstalledTestDescriptorModel` could be renamed to `Descriptor` and `Descriptor` could become a `DescriptorItem` or `DescriptorWithUpdateStatus`